### DISCUSSION TITLE
1750Hz tone missing

### DISCUSSION THREAD

**Discussion by OE1MWW** [egzumer/uv-k5-firmware-custom#79] [Q&A] [answered]
0.15 was great, 0.16 is even better.
But, unfortunately I have a repeater in my area, that wants to hear the 1750Hz tone to activate.
And the 1750Hz option is missing in that great firmware ;-)

**Comment by egzumer** [comment, marked as answer]
https://github.com/egzumer/uv-k5-firmware-custom/wiki/Button-functions#function-button-ii

**Reply by OE1MWW** [reply]
Thanks! Me again, did not read the Wiki ;-)

**Comment by ErikS-web** [comment]
Extra explanation in manual/wiki at [`Radio-operation`](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Radio-operation)  under section [`1750 Hz toneburst for repeater access`](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Radio-operation#1750-hz-toneburst-for-repeater-access)

When the `PTT` is pressed, the 1750 Hz can be activated by pressing [`Function-button-II`](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Button-functions#function-button-ii) (_side key 2, bottom_)
